smac/epm/random_epm.py

```diff
@@ -13,12 +16,50 @@
 class RandomEPM(AbstractEPM):
     """EPM which returns random values on a call to ``fit``."""
 
-    def __init__(self, **kwargs):
+    def __init__(self,
+                 configspace: ConfigurationSpace,
+                 types: np.ndarray,
+                 bounds: np.ndarray,
+                 seed: int,
+                 instance_features: Optional[np.ndarray] = None,
+                 pca_components: Optional[int] = None,
+                 ) -> None:
+        """Constructor
+
+        Parameters
+        ----------
+        configspace : ConfigurationSpace
+            Configuration space to tune for.
+        types : np.ndarray (D)
+            Specifies the number of categorical values of an input dimension where
+            the i-th entry corresponds to the i-th input dimension. Let's say we
+            have 2 dimension where the first dimension consists of 3 different
+            categorical choices and the second dimension is continuous than we
+            have to pass np.array([3, 0]). Note that we count starting from 0.
+        bounds : np.ndarray
+            bounds of input dimensions: (lower, uppper) for continuous dims; (n_cat, np.nan) for categorical dims
+        seed : int
+            The seed that is passed to the model library.
+        instance_features : np.ndarray (I, K)
```

Comment:
Looking at the documentation it is not obvious that instance_features is Optional[int]

Reply:
Good catch.